agent/agent-bootstrap/src/main/java/com/microsoft/applicationinsights/agent/bootstrap/configuration/ConfigurationBuilder.java

```diff
@@ -292,4 +281,21 @@ private void log(Logger logger) {
             logger.warn(message, args);
         }
     }
+
+    public static Configuration loadJsonConfigFile(Path configPath) throws IOException{
+        assert(Files.exists(configPath));
```

Comment:
assert keyword is not used much (since not active by default without `-ea` flag), better to throw exception, e.g.

```suggestion
        if (!Files.exists(configPath)) {
            throw new IllegalStateException("config file does not exist: " + configPath);
        }
```